tools/alive.cpp

```diff
@@ -19,17 +19,18 @@ using namespace std;
 
 
 static void show_help() {
-  cerr <<
-    "Usage: alive2 <options> <files.opt>\n"
-    "Options:\n"
-    " -root-only\tCheck the expression's root only\n"
-    " -v\t\tVerbose mode\n"
-    " -smt-stats\tShow SMT statistics\n"
-    " -smt-to:x\tTimeout for SMT queries in ms\n"
-    " -max-mem:x\tMax memory consumption in MB (aprox)\n"
-    " -smt-verbose\tPrint all SMT queries\n"
-    " -skip-smt\tSkip all SMT queries\n"
-    " -h / --help\tShow this help\n";
+  cerr << "Usage: alive2 <options> <files.opt>\n"
```

Comment:
please avoid committing these random changes with the main patch.

Reply:
yes, @pranavk always review your diffs for stuff like this before submitting

Reply:
ah, this was `clang-format` integrated in my workflow that forced this change; i will fix this.